Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -254,6 +254,11 @@ private void ContentGrid_ManipulationStarted(object sender, ManipulationStartedR
 
                 _contentStoryboard = new Storyboard();
                 _contentStoryboard.Children.Add(_contentAnimation);
+
+                _contentStoryboard.Completed += (s, a) =>
```

Comment:
I'd like to see the event unsubscribed when the control is unloaded. Can then be resubscribed when loaded if the storyboard is not null